### PR TITLE
feat(ingest) - add audit actor urn to auditStamp

### PR DESCRIPTION
**Pull request by neojunjie**
Currently, Unknown actor urn is tagged whenever we ingest new dataset via openapi or restli api. It will be more useful to know who is making the change or ingesting into datahub. It is to reuse authentication context to return actor urn derived from the access token.

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by neojunjie** [comment]
Was trying to do the same for openapi-servlet by changing MappingUtil class. It seemed that I was not able to call authentication context in here?

File: metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java [hunk 1]
```diff
@@ -193,9 +194,11 @@ public Task<Void> ingest(@ActionParam(PARAM_ENTITY) @Nonnull Entity entity,
 
     SystemMetadata systemMetadata = populateDefaultFieldsIfEmpty(providedSystemMetadata);
 
-    // TODO Correctly audit ingestions.
-    final AuditStamp auditStamp =
-        new AuditStamp().setTime(_clock.millis()).setActor(Urn.createFromString(Constants.UNKNOWN_ACTOR));
+    Authentication authentication = AuthenticationContext.getAuthentication();
+    String actorUrnStr = authentication.getActor().toUrnStr();
+    // Getting actor from AuthenticationContext
+    log.debug(String.format("Retrieving AuthenticationContext for Actor with : %s", actorUrnStr));
```
Comment (jjoyce0510) [inline metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java:200]: Let's remove this line!

**Review by jjoyce0510 — COMMENTED** [review]
Overall looks good to me! Minor comments about debug line. Thank you for catching this :)

File: metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java (hunk 1 above)
Comment (neojunjie) [reply metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java]: @jjoyce0510 added actor urn for openapi-servlet as well.

**Review by pedro93 — APPROVED** [review]
LGTM

**Comment by neojunjie** [comment]
Now there is a failed test in EntitiesControllerTest . Not able to get authentication context from the test controller. Any comment what to add?

File: metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java (hunk 1 above)
Comment (jjoyce0510) [reply metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java]: awesome- thank you

**Comment by jjoyce0510** [comment]
It seems we will need to update that test to set a "mock" AuthenticationContext since it's thread-specific - @neojunjie do you need some more specifics?

**Comment by neojunjie** [comment]
> It seems we will need to update that test to set a "mock" AuthenticationContext since it's thread-specific - @neojunjie do you need some more specifics?

I think I am good. Ty for the hint.

**Review by jjoyce0510 — APPROVED** [review]
Awesome PR, @neojunjie  - Thank you!

**Pull request merged**